cartographer_rviz/CMakeLists.txt

```diff
@@ -52,11 +56,9 @@ if(rviz_QT_VERSION VERSION_LESS "5")
 else()
   message(STATUS "Using Qt5 based on the rviz_QT_VERSION: ${rviz_QT_VERSION}")
   find_package(Qt5 ${rviz_QT_VERSION} EXACT REQUIRED Core Widgets)
-  set(QT_LIBRARIES Qt5::Widgets)
+  set(QT_LIBRARIES Qt5::Core Qt5::Widgets)
   include_directories(${Qt5Widgets_INCLUDE_DIRS})
 endif()
-add_definitions(-DQT_NO_KEYWORDS)
-add_library(${PROJECT_NAME} ${ALL_SRCS})
```

Comment:
Do we really need QT_NO_KEYWORDS? If so, consider making it a private definition?

Reply:
Feel free to question and simplify the CMake build, but please in a separate change. This one is already complicated enough.